### PR TITLE
We created the user, questions, answers, and votes models/migrations

### PR DESCRIPTION
**Pull request by kexline4710**

### CODE REVIEW

File: app/models/answer.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Answer < ActiveRecord::Base
+  belongs_to :question
+  belongs_to :responder, class_name: "User", foreign_key: :user_id
```
Comment (ryanbriones) [inline app/models/answer.rb]: Why not responder_id instead of user_id?

File: app/models/question.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class Question < ActiveRecord::Base
+  belongs_to :asker, class_name: "User", foreign_key: :user_id
```
Comment (ryanbriones) [inline app/models/question.rb]: Why not asker_id instead of user_id?

File: app/models/vote.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class Vote < ActiveRecord::Base
+  belongs_to :answer
+  belongs_to :voter, class_name: "User" foreign_key: :user_id
```
Comment (ryanbriones) [inline app/models/vote.rb]: Why not voter_id instead of user_id?

**Comment by mikelikesbikes** [comment]
No tests? Can you think of behavior on your models [that could be tested](https://github.com/thoughtbot/shoulda-matchers#activerecord-matchers)?

**Comment by ryanbriones** [comment]
As a matter of preference, there's a lot of code in here that doesn't relate to one specific feature, but many separate features.

What was the rationale in doing that?

I'd prefer that you were working on one thing at a time (user can add questions, user can add answers to questions, user can add tags to questions)

**Comment by mikelikesbikes** [comment]
@ryanbriones :+1:

**Comment by kexline4710** [comment]
In the initial stages of development, we determined the database schema and routes as a team and then delegated these tasks to two separate pairs. After merging both of our pull requests we plan to work on individual features.

**Comment by mikelikesbikes** [comment]
@kexline4710 interesting. So, add a commit (or 3) to add specs for the associations you defined on your models, and this should be good to go.

**Comment by mikelikesbikes** [comment]
:shipit:

**Pull request merged**